### PR TITLE
👮‍♀️🚔 Add SwiftLint 👮‍♀️🚔

### PR DESCRIPTION
**Pull request by natanrolnik**

### CODE REVIEW

File: KinSDK/KinSDKTests/KinAccountTests.swift [hunk 1]
```diff
@@ -38,12 +42,11 @@ class KinAccountTests: XCTestCase {
             let publicAddress = account?.publicAddress
 
             XCTAssertEqual(publicAddress, expectedPublicAddress)
-        }
-        catch {
+        } catch {
```
Comment (avi-kik) [inline KinSDK/KinSDKTests/KinAccountTests.swift]: If we can't tweak this rule, I will not accept this PR.

Comment (natanrolnik) [reply KinSDK/KinSDKTests/KinAccountTests.swift]: Of course we can. But what’s the problem?

Comment (avi-kik) [reply KinSDK/KinSDKTests/KinAccountTests.swift]: I prefer `catch` to be on a separate line.

Comment (natanrolnik) [reply KinSDK/KinSDKTests/KinAccountTests.swift]: I don't really care, it's more important that we agree and stick to something, than "to be right".
It should be aligned to `if ... else` being on the same line as the closing braces. 
[Check here](https://github.com/realm/SwiftLint/blob/master/Rules.md#statement-position)

I was also going to suggest we follow the snippet that Xcode suggests. This is how it looks:

Swift do-catch snippet

Comment (avi-kik) [reply KinSDK/KinSDKTests/KinAccountTests.swift]: I don't like tools that do not let me choose the formatting.  This is my-way-or-anything-goes, and that's stupid.

Comment (natanrolnik) [reply KinSDK/KinSDKTests/KinAccountTests.swift]: It’s not, we can customize it

**Pull request merged**